server/src/main/java/com/hedera/block/server/BlockNodeApp.java

```diff
@@ -96,7 +96,6 @@ public void start() throws IOException {
                 .build();
 
         // Build the web server
-        // TODO: make port server a configurable value.
```

Comment:
I probably forgot to remove that comment when did it configurable 😅  thank you for taking care of it.